---
string config_path
string save_path
string circled
string circle_filter
string circle_luv
string object_detect
bool   save_to_drone
